upgrade_test.py

```diff
@@ -828,6 +831,133 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
 
         InfoEvent(message='all nodes were upgraded, and last workaround is verified.').publish()
 
+    def test_custom_profile_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
+        """
+        Run a load of a custom profile.
+        Upgrade half of nodes in the cluster, and run rollback.
+        Upgrade all nodes to new version in the end.
+        """
+        InfoEvent(message='Running a prepare load for the initial custom data').publish()
+        if not (prepare_cs_user_profiles := self.params.get('prepare_cs_user_profiles')):
+            SyntaxError("Parameter 'prepare_cs_user_profiles' is not supplied")
+
+        user_profiles, durations = self.parse_cs_user_profiles_param(prepare_cs_user_profiles)
+        stress_before_upgrade, _ = self.run_cs_user_profiles(cs_profiles=user_profiles, duration=durations)
+        for queue in stress_before_upgrade:
+            self.verify_stress_thread(cs_thread_pool=queue)
+
+        # write workload during entire test
+        InfoEvent(message='Starting write workload during entire test').publish()
+        if not (cs_user_profiles := self.params.get('cs_user_profiles')):
+            SyntaxError("Parameter 'cs_user_profiles' is not supplied")
+
+        user_profiles, durations = self.parse_cs_user_profiles_param(cs_user_profiles)
+        entire_write_thread_pool, stress_cmds = self.run_cs_user_profiles(cs_profiles=user_profiles, duration=durations)
+
+        if "scylla-qa-internal" in ",".join(cs_user_profiles):
```

Comment:
if we are using it like that, we should return location of user profiles, not the full stress commands
we shouldn't duplicate the logic we have in `get_profile_content`

Reply:
`get_profile_content` is preformed on runner, `run_cs_user_profiles` - on loader. So location of user profile is not same.

Reply:
so here you need the local file, why do you need this if statement ?
just iterate on the `user_profiles`

the logice around "scylla-qa-internal` should be spread around, it should be only in only place

Reply:
Change performed in https://github.com/scylladb/scylla-cluster-tests/pull/6751